### PR TITLE
doc/user: document service accounts

### PR DESCRIPTION
**Pull request by morsapaes**

### CODE REVIEW

File: doc/user/content/manage/access-control/service-accounts.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+1. In the side navigation bar, click **+ Create New** > **App Password**.
+
+1. In the **New app password** modal, select **Type** > **Service** and name the
+new app password. Under **User**, specify the new service user you'd like to
+create and associate with the new app password.
+
+1. Optionally, associate the new service user with existing **Roles** to grant
+it existing database privileges. You can manage these privileges at any time
+using [RBAC](/manage/access-control/#role-based-access-control-rbac).
```
Comment (benesch) [inline doc/user/content/manage/access-control/service-accounts.md:29]: The roles in the service account dialog aren't the database roles that are described in the RBAC docs. They're just the organization roles: "Organization Member" and "Organization Admin:

image

To grant database roles to the service user via RBAC, you need to log in to the SQL interface and run `GRANT` commands.

File: doc/user/content/manage/access-control/service-accounts.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+   existing database privileges.
+
+  ```hcl
+  resource "materialize_database_grant" "database_usage" {
+    role_name     = materialize_role.production_dashboard.name
+    privilege     = "USAGE"
+    database_name = "production_analytics"
+    region        = "aws/us-east-1"
+  }
+  ```
```
Comment (benesch) [inline doc/user/content/manage/access-control/service-accounts.md:70]: This bit about roles looks right though!

**Review by benesch — COMMENTED** [review]
LGTM, just one note about what the "roles" field in the UI does. Thanks for writing these up!

File: doc/user/content/manage/access-control/service-accounts.md (hunk 1 above)
Comment (morsapaes) [reply doc/user/content/manage/access-control/service-accounts.md:29]: Err, shouldn't have just trusted my memory here and paid attention to the Terraform code — thank you. 🌫️

**Review by sjwiesman — APPROVED** [review]

**Pull request merged**